cookbook/integrations/kubernetes/k8s_spark/pyspark_pi.py

```diff
@@ -41,6 +41,34 @@ def hello_spark(partitions: int) -> float:
            # Regular Pypsark code
            ...
 
+# To run a Spark job on Databricks platform, just add Databricks config to the task config. Databricks Config is is the same as the databricks job request. Refer to `Databricks job request <https://docs.databricks.com/dev-tools/api/2.0/jobs.html#request-structure>`_
+
+   .. code-block:: python
+
+       @task(
+           task_config=Spark(
+               # this configuration is applied to the spark cluster
+               spark_conf={
+                   "spark.driver.memory": "1000M",
+                   "spark.executor.instances": "2",
+                   "spark.driver.cores": "1",
+               },
+               databricks_conf={
+                   "run_name": "flytekit databricks plugin example",
+                   "new_cluster": {
```

Comment:
is there an option for an existing cluster?

Reply:
yes, we can submit to existing cluster